lib/java/test/org/apache/thrift/async/TestTAsyncClient.java

```diff
@@ -1,13 +1,16 @@
 package org.apache.thrift.async;
 
-import junit.framework.TestCase;
 
 import org.apache.thrift.TException;
-
+import org.junit.jupiter.api.Test;
 import thrift.test.Srv;
 import thrift.test.Srv.AsyncClient;
 
-public class TestTAsyncClient extends TestCase {
+import static org.junit.jupiter.api.Assertions.fail;
```

Comment:
This case of using fail() should really use assertThrows(). There may be other similar cases where fail() is used instead of assertThrows().

Reply:
this is fixed now